src/openforms/submissions/admin.py

```diff
@@ -301,7 +306,7 @@ def export_xml(self, request, queryset):
     )
 
     def retry_processing_submissions(self, request, queryset):
-        submissions = queryset.filter(needs_on_completion_retry=True)
+        submissions = queryset.all()
```

Comment:
```suggestion
        submissions = queryset.filter(registration_status=RegistrationStatuses.failed)
```

This will allow us to resend failed registrations (if they are no longer tried automatically or even if they still are).